src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -369,19 +364,22 @@ void CascadiaSettings::_ReorderProfilesToMatchUserSettingsOrder()
 // - <none>
 void CascadiaSettings::_RemoveHiddenProfiles()
 {
-    // remove_if will move all the profiles where the lambda is true to the end
-    // of the list, then return a iterator to the point in the list where those
-    // profiles start. The erase call will then remove all of those profiles
-    // from the list. This is the [erase-remove
-    // idiom](https://en.wikipedia.org/wiki/Erase%E2%80%93remove_idiom)
-    _profiles.erase(std::remove_if(_profiles.begin(),
-                                   _profiles.end(),
-                                   [](auto&& profile) { return profile.Hidden(); }),
-                    _profiles.end());
+    for (uint32_t i{}; i < _profiles.Size();)

```

Comment:
Oh, please just assign 0 to i. I know the `{}` will technically default construct it to 0, but that's so much more obtuse than the thing everyone recognizes.

Reply:
ha! you know, i thought you were part of our discussions about zero-initializing things with `{}`, and perhaps with the ayes and not the nays